spec/ferver_spec.rb

```diff
@@ -99,8 +99,10 @@
   end
 
   context 'given a list of files' do
+    let(:file_1) { double('file', name: 'file1') }
+    let(:file_2) { double('file', name: 'file2') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.